andela/AndelaEatsUI
# #164296390 Admin should view admin user list

#### What does this PR do?

- Implements viewing all available admin users.

#### Description of Task to be completed?

- Add action to fetch admin users.
- Add revoke admin rights functionality
- Add styling to the admin users table

#### How should this be manually tested?

- Log into the application and navigate to the admin dashboard
- Click on the user's tab on the side Nav
- You should see a list of all admin users.

#### What are the relevant pivotal tracker stories?

- [#164296390](https://www.pivotaltracker.com/story/show/164296390)

#### Background Context
N/A

#### Screenshots (If Applicable)

<img width="1645" alt="screen shot 2019-03-04 at 10 05 37" src="https://user-images.githubusercontent.com/35847046/53716777-0badbd80-3e67-11e9-8eee-f497aa326312.png">
